### ISSUE TITLE
Feature request - MOTD for namelayer groups.

### ISSUE THREAD

**Issue by Convoy20** [Civcraft/NameLayer#238]
Similar to how when you get a message when you log in for the first time after being added to a group while being offline, allow group owners to set a message of the day notification to a namelayer group, to alert any members when they log in next to a message with updates or information.

**Comment by ProgrammerDan** [comment]
Yeah, that'd be a cool feature. Could get noisy, so would want to implement
it with some governance, if you've got more then 2-3 messages, show a
single "you have waiting group MOTDs" that opens a menu where you can read
them all. Otherwise, just show the messages.

Might just need in general a menu system for it, to allow retrieving old
MOTDs. Also, I'd assume you'd want to flag that a MOTD was viewed for a
particular person, so that by default you don't get re-notified of an
unchanged MOTD.
